Pixel Game/Assets/Scripts/Health.cs

```diff
@@ -65,6 +68,9 @@ public void TakeDamage(int damage, Vector2 sourcePoint)
         if (hurtSound != null)
             AudioSource.PlayClipAtPoint(hurtSound, this.gameObject.transform.position);
 
+        screenShake.StartScreenShake();
+        Debug.Log("We shake screena");
```

Comment:
Not sure it's a good idea to keep logging things if you have already confirmed that it works. It will clutter up the concole and make it harder to debug future features/bugs.

Reply:
You are correct, i missed that!